desktop_projects/pybind_test/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_library(stm32-avionics-py SHARED 
```

Comment:
Adding pybind is probably out of scope for this PR. Not that it's bad, but there's already a lot here so it'd probably be worth separating it for a future change.